k8s/testing.yaml

```
@@ -114,6 +114,8 @@ spec:
                 secretKeyRef:
                   name: teamy
                   key: internal-service-account-api-token
+            - name: TEAMY_WEB_UI_BASE_URL
+              value: "http://localhost:3000"
```

```suggestion
              value: "https://testing.teamyapp.com"
```
Teamy backend API in testing environment should redirect user to testing UI.